scipy/signal/fir_filter_design.py

```diff
@@ -578,23 +599,31 @@ def remez(numtaps, bands, desired, weight=None, Hz=1, type='bandpass',
 
     Examples
     --------
-    We want to construct a filter with a passband at 0.2-0.4 Hz, and
-    stop bands at 0-0.1 Hz and 0.45-0.5 Hz. Note that this means that the
-    behavior in the frequency ranges between those bands is unspecified and
-    may overshoot.
+    For a signal sampled at 100 Hz, we want to construct a filter with a
+    passband at 20-40 Hz, and stop bands at 0-10 Hz and 45-50 Hz. Note that
+    this means that the behavior in the frequency ranges between those bands
+    is unspecified and may overshoot.
 
     >>> from scipy import signal
-    >>> bpass = signal.remez(72, [0, 0.1, 0.2, 0.4, 0.45, 0.5], [0, 1, 0])
+    >>> fs = 100
+    >>> bpass = signal.remez(72, [0, 10, 20, 40, 45, 50], [0, 1, 0], fs=fs)
     >>> freq, response = signal.freqz(bpass)
-    >>> ampl = np.abs(response)
 
     >>> import matplotlib.pyplot as plt
-    >>> fig = plt.figure()
-    >>> ax1 = fig.add_subplot(111)
-    >>> ax1.semilogy(freq/(2*np.pi), ampl, 'b-')  # freq in Hz
+    >>> plt.semilogy(0.5*fs*freq/np.pi, np.abs(response), 'b-')
+    >>> plt.grid(alpha=0.25)
+    >>> plt.xlabel('Frequency (Hz)')
+    >>> plt.ylabel('Gain')
     >>> plt.show()
 
     """
+    if Hz is None and fs is None:
+        fs = 1.0
+    elif Hz is not None:
+        if fs is not None:
+            raise ValueError("Values cannot be given for both 'Hz' and 'fs'.")
+        fs = Hz
```

Comment:
It might be worth making a little private function to handle this:
```
def _check_fs(fs, other, name="nyq", default=1.):
   if other is None and fs is None:
       fs = default
   elif other is not None:
       if fs is not None:
           raise ValueError("Values cannot be given for both '%s' and 'fs'." % (name,))
       fs = other
   return fs
```
That way when we do decide to deprecate, we can do it in one place

Reply:
I ended up using a simplified version of this for the change 'nyq' -> 'fs'.  Modifying it to also handle the single case of  'Hz' -> 'fs' didn't seem worthwhile.